lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -1415,6 +1415,9 @@ fn create_config(params: &ConnectParams, pin: String) -> Config {
         enable_tls: true,
         // For now, NLA is opt-in via an environment variable.
         // We'll make it the default behavior in a future release.
+        //
+        // TODO(zmb3, probakowki): keep this logic synchronized with
+        // lib/srv/desktop/audit.go for proper auditing and telemetry
```

Comment:
Should we move the logic completely to Go and just pass it as parameter to Rust? That would ensure consistency